bats/tests/profile/create-profile-output.bats

```diff
@@ -0,0 +1,220 @@
+load '../helpers/load'
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start app' {
+    start_container_engine
+    wait_for_container_engine
+}
+
+@test 'report invalid parameters for plist' {
+    run rdctl create-profile --output=plist --from-settings --hive=fish
+    assert_failure
+    assert_output --partial "registry hive and type can't be specified with plist"
+}
+
+@test 'report unrecognized output-options' {
+    run rdctl create-profile '--output=pickle'
+    assert_failure
+    assert_output --partial $'received unrecognized \'--output FORMAT\' option of pickle; "plist" or "reg" must be specified'
+}
+
+@test 'report unrecognized registry sub-options' {
+    run rdctl create-profile --output=reg --hive=hklm --type=ruff --from-settings
+    assert_failure
+    assert_output --partial "invalid registry type of 'ruff' specified"
+}
+
+@test 'generates registry output for hklm/defaults' {
+    run rdctl create-profile --output reg --from-settings
+    assert_success
+    assert_output --partial '[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Rancher Desktop\defaults\application]'
+
+    run rdctl create-profile --output reg --hive=hklm --from-settings
+    assert_success
+    assert_output --partial '[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Rancher Desktop\defaults\application]'
+
+    run rdctl create-profile --output reg --hive=HKLM --type=Defaults --from-settings
+    assert_success
+    assert_output --partial '[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Rancher Desktop\defaults\application]'
+
+    run rdctl create-profile --output reg --type=DEFAULTS --from-settings
+    assert_success
+    assert_output --partial '[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Rancher Desktop\defaults\application]'
+}
+
+@test 'generates default registry output from inline json' {
+    run rdctl create-profile --output reg --body '{"application": { "window": { "quitOnClose": true }}}'
+    assert_success
+    assert_output --partial '[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Rancher Desktop\defaults\application\window]'
+    assert_output --partial '"quitOnClose"=dword:1'
+}
+
+@test 'generates registry output for hklm/locked' {
+    run rdctl create-profile --output reg --hive=Hklm --type=Locked --from-settings
+    assert_success
+    assert_output --partial '[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Rancher Desktop\locked\application]'
+    run rdctl create-profile --output reg --type=LOCKED --from-settings
+    assert_success
+    assert_output --partial '[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Rancher Desktop\locked\application]'
+}
+
+@test 'generates registry output for hkcu/defaults' {
+    run rdctl create-profile --output reg --hive=Hkcu --from-settings
+    assert_success
+    assert_output --partial '[HKEY_CURRENT_USER\SOFTWARE\Policies\Rancher Desktop\defaults\application]'
+    run rdctl create-profile --output reg --hive=hkcu --type=Defaults --from-settings
```

Comment:
Put a blank line before `run` command, if there are several in a single `@test`. Happens in a few more tests below.

Reply:
yup